python/fusion_engine_client/analysis/analyzer.py

```diff
@@ -230,7 +230,7 @@ def _plot_data(name, idx, marker_style=None):
                 # If there's no data, draw a dummy trace so it shows up in the legend anyway.
                 map_data.append(go.Scattermapbox(lat=[np.nan], lon=[np.nan], name=name, **style))
 
-        _plot_data('RTK Fixed', solution_type == SolutionType.RTKFixed, {'color': 'red'})
+        _plot_data('RTK Fixed', solution_type == SolutionType.RTKFixed, {'color': 'orange'})
```

Comment:
if we're changing the color, should we consider making the size consistent? (5)

Reply:
Sure, we can do that

Reply:
Went with 8 for both instead of 5. The 5 dots are a little small on the max zoom level supported by plotly.